agent/startup.sh

```diff
@@ -103,6 +103,12 @@ fi
 
 export CATTLE_AGENT_IP=${CATTLE_AGENT_IP:-$DETECTED_CATTLE_AGENT_IP}
 
+if [ -n "$CATTLE_PHYSICAL_HOST_UUID" ]; then
```

Comment:
I'd prefer the Python treat blank as unset.  I've ran into bugs with that before.  I would hope that wouldn't cause a major issue